examples/autopilot_server/autopilot_server.cpp

```diff
@@ -0,0 +1,312 @@
+#include <future>
+
+#include <iostream>
+#include <thread>
+#include <mavsdk/mavsdk.h>
+#include <mavsdk/plugins/mavlink_passthrough/mavlink_passthrough.h>
+#include <mavsdk/plugins/telemetry/telemetry.h>
+#include <mavsdk/plugins/action/action.h>
+#include <mavsdk/plugins/param_server/param_server.h>
+#include <mavsdk/plugins/param/param.h>
+#include <mavsdk/plugins/telemetry_server/telemetry_server.h>
+#include <mavsdk/plugins/action_server/action_server.h>
+#include <mavsdk/plugins/mission_raw_server/mission_raw_server.h>
+#include <mavsdk/plugins/mission/mission.h>
+
+/*
+ This example runs a MAVLink "autopilot" utilising the MAVSDK server plugins
+ on a seperate thread. This uses two MAVSDK instances, one GCS, one autopilot.
+
+ The main thread acts as a GCS and reads telemetry, parameters, transmits across
+ a mission, clears the mission, arms the vehicle and then triggers a vehicle takeoff.
+
+ The autopilot thread handles all the servers and triggers callbacks, publishes telemetry,
+ handles and stores parameters, prints received missions and sets the vehicle height to 10m on
+ successful takeoff request.
+*/
+
+using namespace mavsdk;
+
+using std::chrono::seconds;
+using std::chrono::milliseconds;
+using std::this_thread::sleep_for;
+
+#define ERROR_CONSOLE_TEXT "\033[31m" // Turn text on console red
+#define TELEMETRY_CONSOLE_TEXT "\033[34m" // Turn text on console blue
+#define NORMAL_CONSOLE_TEXT "\033[0m" // Restore normal console colour
```

Comment:
I removed that stuff from the examples a while ago. In my opinion it adds noise and distracts from the actual example.